packages/components/alert-dialog/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'react'
+
+import { AlertDialog as Root, AlertDialogProps } from './AlertDialog'
+import { AlertDialogAction } from './AlertDialogAction'
+import { AlertDialogBody } from './AlertDialogBody'
+import { AlertDialogCancel } from './AlertDialogCancel'
+import { AlertDialogContent } from './AlertDialogContent'
+import { AlertDialogDescription } from './AlertDialogDescription'
+import { AlertDialogFooter } from './AlertDialogFooter'
+import { AlertDialogHeader } from './AlertDialogHeader'
+import { AlertDialogOverlay } from './AlertDialogOverlay'
+import { AlertDialogPortal } from './AlertDialogPortal'
+import { AlertDialogTitle } from './AlertDialogTitle'
+import { AlertDialogTrigger } from './AlertDialogTrigger'
+
+export * from './AlertDialog'
+export { useAlertDialog } from './AlertDialogContext'
+export { type AlertDialogActionProps } from './AlertDialogAction'
+export { type AlertDialogBodyProps } from './AlertDialogBody'
+export { type AlertDialogCancelProps } from './AlertDialogCancel'
+export { type AlertDialogContentProps } from './AlertDialogContent'
+export { type AlertDialogDescriptionProps } from './AlertDialogDescription'
+export { type AlertDialogFooterProps } from './AlertDialogFooter'
+export { type AlertDialogHeaderProps } from './AlertDialogHeader'
+export { type AlertDialogOverlayProps } from './AlertDialogOverlay'
+export { type AlertDialogPortalProps } from './AlertDialogPortal'
+export { type AlertDialogTitleProps } from './AlertDialogTitle'
+export { type AlertDialogTriggerProps } from './AlertDialogTrigger'
+
+export const AlertDialog: FC<AlertDialogProps> & {
```

Comment:
Don't you also need to set the `displayNames` before exporting the compound, like we do in the Dialog ?
```
Trigger.displayName = 'Dialog.Trigger'
Portal.displayName = 'Dialog.Portal'
Overlay.displayName = 'Dialog.Overlay'
Content.displayName = 'Dialog.Content'
CloseButton.displayName = 'Dialog.CloseButton'
Title.displayName = 'Dialog.Title'
Description.displayName = 'Dialog.Description'
```

Reply:
Yes :( this is needed only for storybook D: I think that we should avoid defining the `displayName` two times

Reply:
I agree but it is important the the component's names are correct when a user click on "show source"